version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.2.0"
+ThisBuild / version := "0.2.1"
```

Comment:
Bumping the release version since I released v0.2.0 already and this is adding patch/changes to nullable fields.